### PR TITLE
feat(helix): allow to extend config from kurtosis config file

### PR DESCRIPTION
**Pull request by thedevbirb**
This PR allows using the `kurtosis_config.yaml` file as follows:
```yaml
mev_params:
    # ...
    helix_relay_config_extension:
    	constraints_api_config:
      		check_constraints_signature: false
      		max_block_value_to_verify_wei: "0x00000000000000000000000000000000000000000000000000000000deadbeef"
     # ...
```

### CODE REVIEW

File: src/mev/mev_relay/helix_launcher.star [hunk 1]
```diff
@@ -162,6 +163,8 @@ def launch_helix_relay(
         ),
     )
 
+    plan.print(json.indent(json.encode(helix_config_template_data)))
```
Comment (merklefruit) [inline src/mev/mev_relay/helix_launcher.star:166]: is this useful?

File: src/mev/mev_relay/helix_launcher.star [hunk 2]
```diff
@@ -178,27 +181,34 @@ def new_config_template_data(
...
+        "network_config": {
             "dir_path": network_config_dir_path,
             "genesis_validator_root": genesis_validator_root,
             "genesis_time": genesis_time,
         },
     }
+
+    if config_extension != None:
+        for key, value in config_extension.items():
+            config_hashmap[key] = value
```
Comment (merklefruit) [inline src/mev/mev_relay/helix_launcher.star:212]: nice

**Review by merklefruit — APPROVED** [review]
great!

File: src/mev/mev_relay/helix_launcher.star (hunk 1 above)
Comment (thedevbirb) [reply src/mev/mev_relay/helix_launcher.star:166]: Yes it shows a nice print with the config which Helix is trying to run. Other services do that as well and I think it's useful.

**Pull request merged**